src/components/solidLabel/SolidLabel.tsx

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import { SSolidLabel } from "./styles/SSolidLabel";
+
+export type ISolidLabelType = "default" | "important" | "green" | "red" | "yellow" | "grey" | "greyOutline";
```

Comment:
@MoritzGruber is this the coding we are using for this?

Reply:
yes i am fine with this its easy to use, with enum you need to import the enum as well otherwise -